environment/interpretation/lane_detector/data/Samples.model

```diff
@@ -1,42 +1,35 @@
-																											svm_type c_svc
+svm_type c_svc
 kernel_type polynomial
-degree 4
+degree 3
 gamma 0.00520833
 coef0 0
 nr_class 2
-total_sv 31
-rho -0.792476
+total_sv 24
+rho -0.941211
```

Comment:
Yup

regards,
Krishna Bagadia
Department of Computer Science and Enginnering
Autonomous Ground Vehicle
IIT Kharagpur.
Email id: krishna.bagadia2@gmail.com
contact no. : 07872685359
www.agv.iitkgp.ac.in

On Fri, May 8, 2015 at 11:24 AM, Abinash Meher <notifications@github.com>
wrote:

> In environment/interpretation/lane_detector/data/Samples.model
> <https://github.com/AGV-IIT-KGP/freezing-batman/pull/137#discussion_r29918367>
> :
>
> >  gamma 0.00520833
> >  coef0 0
> >  nr_class 2
> > -total_sv 31
> > -rho -0.792476
> > +total_sv 24
> > +rho -0.941211
>
> Are these files generated by code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AGV-IIT-KGP/freezing-batman/pull/137/files#r29918367>.
>